snippets/man_gen/hubl_custom_module_fields.json

```diff
@@ -486,5 +486,111 @@
       "}"
     ],
     "description": "HubSpot Font Field"
+  },
+  "Style Tab Group": {
+    "prefix": "styleTab.group",
+    "body": [
+      "{",
+      "\"name\": \"${1:style}\",",
+      "\"label\": \"${2:Style}\",",
+      "\"children\": [${3}],",
+      "\"type\": \"group\",",
+      "\"tab\": \"STYLE\"",
+      "}"
+    ],
+    "description": "Style Tab Field Group"
+  },
+  "Alignment Field": {
+    "prefix": "field.alignment",
+    "body": [
+      "{",
+      "\"name\": \"${1:alignment_field}\",",
+      "\"label\": \"${2:Alignment field}\",",
+      "\"required\": false,",
+      "\"locked\": false,",
+      "\"type\": \"boolean\",",
```

Comment:
Would it make sense to use the alignment field vs. a custom boolean field for this one (e.g. https://developers.hubspot.com/en/docs/cms/building-blocks/module-theme-fields#alignment)?

Reply:
Good catch!